sailfishusermanager.cpp

```diff
@@ -0,0 +1,245 @@
+/*
+ * Copyright (c) 2020 Jolla Ltd.
+ * Copyright (c) 2020 Open Mobile Platform LLC.
+ *
+ * All rights reserved.
+ *
+ * BSD 3-Clause License, see LICENSE.
+ */
+
+#include "sailfishusermanager.h"
+#include "usermanager_adaptor.h"
+#include "libuserhelper.h"
+
+#include <QDBusConnection>
+#include <QTimer>
+#include <QFile>
+#include <QDir>
+
+#include <pwd.h>
+#include <grp.h>
+#include <unistd.h>
+#include <sys/types.h>
+#include <sys/stat.h>
+
+const char *DBUS_INTERFACE = "org.sailfishos.usermanager";
+const char *DBUS_OBJECT_PATH = "/";
+const char *USER_GROUP = "users";
+const char *GROUPS_IDS_FILE = "/usr/lib/setup/group.ids";
+const char *GROUPS_SYSTEM = "SYSTEM_GROUPS";
+const char *GROUPS_USER = "USER_GROUPS";
+const char *SKEL_DIR = "/etc/skel";
+const char *USER_HOME = "/home/%1";
```

Comment:
You always create a QString out of this. Couldn't it just be a QStringLiteral here?

Reply:
Fixed